activesupport/lib/active_support/deprecation/reporting.rb

```diff
@@ -37,6 +41,44 @@ def silence(&block)
         @silenced_thread.bind(true, &block)
       end
 
+      # Allow previously disallowd deprecation warnings within the block.
```

Comment:
There's a typo here which ought to be "disallowed".

Reply:
Thanks. Fixed in https://github.com/rails/rails/pull/37940/commits/df0959bd3e4a852d0d4ac7569227be5ada16c251